conf/application.conf

```diff
@@ -92,6 +92,12 @@ microservice {
       host     = localhost
       port     = 9025
     }
+
+    email {
```

Comment:
do we need changes to the service profiles or the app.config for different environments? @ChrisCookOC is doing the testing spike so might be able to help with this

Reply:
Will need a change on app-config-base to point to the service properly.

May need further changes for testing didn't get that far today :(

Reply:
I created a PR for this https://github.com/hmrc/app-config-base/pull/10985